src/resource/main.cpp

```diff
@@ -94,18 +94,18 @@ int main(int argc, char** argv)
 #endif
     include_paths.push_back({NGRAPH_HEADERS_PATH, {}, true});
     include_paths.push_back({CLANG_BUILTIN_HEADERS_PATH, {}, true});
-    include_paths.push_back({"/Library/Developer/CommandLineTools/usr/include/c++/v1", {}});
+// include_paths.push_back({"/Library/Developer/CommandLineTools/usr/include/c++/v1", {}});
```

Comment:
Was this change intentional?

Reply:
Yes. No longer embedding system headers in the resource file, just the headers that are from the ngraph directory. The system headers still exist after you delete the repo directory but the ngraph headers don't so they must be preserved. We COULD "install" the ngraph headers and solve the problem that way.